Jenkinsfile

```diff
@@ -31,11 +32,11 @@ pipeline {
       }
     }
 
-    stage ("Run Integration Tests") {
+   stage ("Run Integration Tests on oss") {
       steps {
         script {
           def tasks = [:]
-          ["oss", "dap"].each { deployment ->
+          ["oss"].each { deployment ->
```

Comment:
if you have only one object in the list you can get rid of it and use it hard-coded.

Reply:
i agree with you but  i wanted to make it fixable if we want to add another environment in the future.

Reply:
which other environment will we add? We can go to git history and get the syntax from there. 
The motivation for the separation is so there are no parallel jobs running on the same OCP/K8S environment? I believe we should add a comment on that in the Jenkinsfile itself as it's quite a big decision.

Reply:
o.k i will change it to hard-coded i dont see a big issue here..

Reply:
Thanks.

Reply:
👍